include/emqx.hrl

```diff
@@ -161,6 +161,20 @@
           by     :: binary(),
           desc   :: binary(),
           until  :: integer()
-        }).
+       }).
+
+%%--------------------------------------------------------------------
+%% Flapping
+%%--------------------------------------------------------------------
+
+-record(flapping, {
```

Comment:
why this record is put to a hrl?
it seems to be only used in emqx_flapping module.

Reply:
you may have more to add to this PR, but from my superficial understanding of the current code, it's better to define it as an opaque structure for emqx_flapping